examples/raysgd_nyctaxi.py

```diff
@@ -61,7 +70,7 @@ def __init__(self, cols):
         self.bn4 = nn.BatchNorm1d(16)
 
     def forward(self, *x):
-        x = torch.cat(x, dim=1)
+        x = x[0]
```

Comment:
what was the data shape? does this lose input data?

Reply:
I modified parameter "*x" -> "x", then the type of x is `torch.Tensor` and the shape is `[batchsize, len(features)]`.